meinberlin/apps/votes/models.py

```diff
@@ -56,14 +58,59 @@ def get_or_create_salt_for_module(module):
         return token_salt.salt
 
 
+class TokenPackage(models.Model):
+    module = models.ForeignKey(Module, on_delete=models.CASCADE)
+    package_number = models.PositiveIntegerField()
+    size = models.PositiveIntegerField()
+    downloaded = models.BooleanField(default=False)
+
+    class Meta:
+        unique_together = ["module", "package_number"]
+
+    @staticmethod
+    def get_sum_token(module):
+        num_tokens = (
+            TokenPackage.objects.filter(module=module)
+            .aggregate(Sum("size"))
+            .get("size__sum")
+        )
+        if num_tokens is None:
+            num_tokens = 0
+        return num_tokens
+
+    @staticmethod
+    def create_next_package(module, size):
+        for i in range(100):
```

Comment:
I dont really get why we need this here? Is it because it might happen that between calculating package_number and actually creating it, there might have already been another one created?